sample-input/benchmarks/LRA/LRA.py

```diff
@@ -1,10 +1,13 @@
 import openmoc
+import openmoc.plotter as plotter
+from openmoc.options import Options
+from lattices import lattices, universes, cells
```

Comment:
it doesn't look like you use `lattices` or `cells` in this script, can they be removed from this import?